### ISSUE TITLE
Quotes and other punctuation not removed from title for files names.

### ISSUE THREAD

**Issue by JamieKnight** [daeken/Benjen#7]
While experimenting with my own content i have found that invalid characters find their way into file names which in turn results in broken links.

The following file:

```
#title 2012 11" MacBook Air
#date 2012-03-14

Post body
```

Produces a file named: `2012-03-14_2012_11"_MacBook_Air.html`

That file is linked to as:

```
<h1 class="post-title"><a href="2012-03-14_2012_11"_MacBook_Air.html">2012 11" MacBook Air</a></h1>
```

The " character then breaks the HTML link.

I am looking at a solution now. Will put together a pull request once i have figure out how.

Cheers,

Jamie

**Comment by daeken** [comment]
Fixed with commit c542c5cd5fc0b785f8937ca7af7d5d03c6c9cde7.  Thanks!

**Comment by JamieKnight** [comment]
Thanks, speedy fix.
